src/librustc/session/config.rs

```diff
@@ -1082,8 +1082,6 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
         "gather borrowck statistics"),
     no_landing_pads: bool = (false, parse_bool, [TRACKED],
         "omit landing pads for unwinding"),
-    debug_llvm: bool = (false, parse_bool, [UNTRACKED],
```

Comment:
Is the idea behind removing this flag that one can pass `-Cllvm-args=-debug` instead? I'm fine with that, but it's not exactly "unused".

Reply:
Oh yes, now I see that the commit message mentions precisely that. As I said I'm fine with it but it's kind of a visible change so I don't feel comfortable giving r+ myself.